### ISSUE TITLE
workspace Folder

### ISSUE THREAD

**Issue by FlorianPydde** [microsoft/dstoolkit-devcontainers#14]
When notebook or src folder in a new container, should we also change the _workspaceFolder_ in the devcontainer.json config when setting up the project? 

Currently it's set to: "workspaceFolder": "/workspaces/dstoolkit-devcontainers)/notebooks"

**Comment by fujikosu** [comment]
@FlorianPydde Yes that part needs to be updated. You can check this section of README.md regarding how you need to change it. https://github.com/microsoft/dstoolkit-devcontainers#how-to-create-a-new-directory-under-src-with-a-new-environment

I just merged new readme.md and new devcontainers version that has no dependency on docker-compose anymore with this https://github.com/microsoft/dstoolkit-devcontainers/pull/13 so you might have missed the update to README.md there.

**Comment by FlorianPydde** [comment]
Fair enough. Since I was starting some EDA I was only looking at the notebook folder not the src one. Should we always copy the example folder in src as starting point ?

**Comment by FlorianPydde** [comment]
Yes, that's what I tried to do, i.e. simply use the notebook folder. But that's when I saw that devcontainer.json "workspaceFolder" key had to be updated, and this is not mentioned in the documentation.

**Comment by fujikosu** [comment]
You don't need to update workspaceFolder key when you use notebooks directory.

the latest main got this below
```
    "workspaceFolder": "/workspaces/dstoolkit-devcontainers/notebooks",
```
and this should work out of the box. Build succeeded on my local and succeeded on David's local too during his PR review. 

1. What specific issue are you getting?
2. Are you using the latest main? If not, I'd suggest you to pull in the latest main to your local and run the command `DevContainers: Rebuild Container Without Cache`
